_posts/2020-09-09-#18267.md

```diff
@@ -10,14 +10,46 @@ status: upcoming
 commit:
 ---
 
-_Notes and questions to follow soon!_
+This pull request implements the portion of
+[BIP325](https://github.com/bitcoin/bips/blob/master/bip-0325.mediawiki)
+responsible for connecting to a signet network and validating blocks (it does
+not implement the code required by miners to sign signet blocks). We are
+touching consensus-critical code today! Let's make sure we do it right :-)
+
 
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+- kallewoof has an (outdated)
+[gist](https://gist.github.com/kallewoof/98b6d8dbe126d2b6f47da0ddccd2aa5a)
+describing how to connect to signet once the client is built. If you can run the
+branch and generate a signet address, you can request a payout from his
+[faucet](https://signet.bc-2.jp/)
+
+- At the start of this meeting I'll ask everyone for their signet address and
+reward participants who answer questions correctly or ask their own good questions!
+
 ## Questions
--->
 
+1. Why do we need test networks and what problems (if any) do we have with the current testnet?
+
+2. What are some of the parameters that distinguish mainnet, testnet and signet?
+Which of these do the different networks have in common? What makes signet special?
+
+3. How many signets are there? Could we start our own just for review club, and how?
```

Comment:
```suggestion
3. How many signets are there? Could we start our own just for the review club, and if so, how?
```

Reply:
taken the "if so,". I think it's fine to refer to 'review club' without the article.